agent/src/agent/pipeline/config/jython_scripts/elastic_loader.py

```diff
@@ -0,0 +1,117 @@
+global sdc
+
+try:
+    sdc.importLock()
+    import sys
+    import os
+
+    sys.path.append(os.path.join(os.environ['SDC_DIST'], 'python-libs'))
+    import requests
+    import traceback
+    import time
+    import urllib
+    import re
+    import json
+    from datetime import datetime
+finally:
+    sdc.importUnlock()
+
+entityName = ''
+N_REQUESTS_TRIES = 3
+
+
+def get_now_with_delay():
+    return int(time.time()) - int(sdc.userParams['DELAY_IN_MINUTES']) * 60
+
+
+def get_backfill_offset():
+    if sdc.lastOffsets.containsKey(entityName):
+        return int(float(sdc.lastOffsets.get(entityName)))
+    if sdc.userParams['INITIAL_TIMESTAMP']:
+        return int(sdc.userParams['INITIAL_TIMESTAMP'])
+    return get_now_with_delay() - get_interval()
+
+
+def get_interval():
+    return int(sdc.userParams['INTERVAL'])
+
+
+def make_request(url_, params={}):
+    for i in range(1, N_REQUESTS_TRIES + 1):
+        try:
+            sdc.log.debug(url_)
+            res = requests.post(
+                url_,
+                json=params,
+                stream=True,
+                headers={'Content-Type': 'application/json'},
+                # timeout=sdc.userParams['QUERY_TIMEOUT']
+            )
+            res.raise_for_status()
+        except requests.HTTPError as e:
+            requests.post(sdc.userParams['MONITORING_URL'] + str(e.response.status_code))
+            sdc.log.error(str(e))
+            if i == N_REQUESTS_TRIES:
+                raise
+            time.sleep(i**2)
+            continue
+        break
+    return res
+
+
+def process_batch(result_, end_):
+    batch = sdc.createBatch()
+    for res in result_['hits']['hits']:
+        # res['sort'] = [res['_source']['timestamp_unix_ms']]
+        del res['_score']
```

Comment:
why are you deleting this? not all search results might return this and in case this field doesn't exist there's gonna be an exception